website/docs/installation/weave-gitops.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Weave GitOps
+sidebar_position: 2
+hide_title: true
+pagination_next: "getting-started"
+---
+
+## Installing Weave GitOps
+
+#### Install Flux
```

Comment:
There is already a section with the exact same name under "Before you begin" so I think what we should do is move the content of this section to the other one.